packages/webshot/install

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -x
+set -e
+
+sudo apt-get update
+sudo apt-get install -y phantomjs
+phantomjs --version
```

Comment:
Please remove

Reply:
Just found the "apt-get" will install the old version (1.9.1) on "trusty" and will crush on xenial. Both of them cannot work well with webshot. 

I might have to say goodbye to this feature and I have spent so much time on this.  All of our customers are professional subscribers of shinyapps.io but we just cannot get the webshot work on it. The original purpose is to generate a nice PDF report from shiny app directly.